gulpfile.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env node */
+'use strict';
+
+const fs = require('fs');
+const fse = require('fs-extra');
+const gulp = require('gulp');
+const file = require('gulp-file');
+const path = require('path');
+const tar = require('tar-fs');
+const stagingWorkflow = require('bin-minify').stagingWorkflow;
+
+const {
+    GIT_BIN_PATH,
+    MIN_PACK_PATH,
+    MIN_PACK_FILENAME,
+} = require('./config');
+
+const GIT_FILENAME = 'git-2.13.5.tar';
+
+gulp.task('default', async (done) => {
+    fse.ensureDirSync(GIT_BIN_PATH);
+
+    await fs.createReadStream(path.join(__dirname, GIT_FILENAME))
+        .pipe(tar.extract(GIT_BIN_PATH))
```

Comment:
This is where all the `bin/` files get generated, correct?

As long as that's true and none of them have manual changes, we should remove them from git, and git ignore them. You'll then also need to create an empty `.npmignore` file (otherwise git ignoring them will stop npm publishing them). That way, they're always included when this package is published, but we don't have to have all the extra files living in the repo itself. Makes the whole thing much easier to understand I think. Any reason not to do that?

Reply:
done